KeyMapCheatSheet.md

```diff
@@ -0,0 +1,93 @@
+# KeyMap Cheat-Sheet
+
+For anyone who is just starting with this neovim config!
+
+##### Note:
+
+- Vim Commands still work as they do one vanilla neovim
```

Comment:
one -> in? on?

Reply:
ion?

Reply:
> ion?

Talking about the grammar I think, in the line `they do one vanilla neovim`, should be `they do in/on neovim`.